pkg/interactive/autocomplete.go

```diff
@@ -94,7 +95,7 @@ func sanitiseTableName(strToEscape string) string {
 	tokens := helpers.SplitByRune(strToEscape, '.')
 	escaped := []string{}
 	for _, token := range tokens {
-		if strings.ContainsAny(token, " -") {
+		if strings.ContainsAny(token, " -") || utils.ContainsUpper(token) {
```

Comment:
out of interest, why are we checking foe `" ="`

Reply:
add comment
`If string contains upper case chars, escape it as Postgres by default converts to lower case'